device-animation-test-rule/build.gradle

```diff
@@ -1,59 +1,58 @@
 apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
 
-group='com.github.VictorAlbertos'
-version='0.0.2'
+group = 'com.github.VictorAlbertos'
+version = '0.0.3'
 
 android {
-  compileSdkVersion 24
-  buildToolsVersion "24.0.1"
-
-  defaultConfig {
-    minSdkVersion 16
-    targetSdkVersion 24
-    versionCode 1
-    versionName "1.0"
-    archivesBaseName = "DeviceAnimationTestRule"
-  }
-  buildTypes {
-    release {
-      minifyEnabled false
-      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+    compileSdkVersion 29
+
+    defaultConfig {
+        minSdkVersion 21
```

Comment:
Is the raise of the `minSdkVersion` required at this point. If not I suggest postponing it to another pull request and release an AndroidX version with `minSdkVersion 16` first to make it more comfortable for users to update their projects.

Reply:
The minSdkVersion declared in androidx.test.uiautomator:uiautomator:2.2.0 is 18 and can't be smaller. But 18 version should be fine too

Reply:
18 sounds better. I guess uiautomator cannot be jetified temporarily to be able to provide a minSdkVersion 16 release?